tests/MongoDB.Analyzer.Tests/Infrastructure/DiagnosticTestCase.cs

```diff
@@ -20,7 +20,10 @@ namespace MongoDB.Analyzer.Tests.Infrastructure;
 [Serializable]
 public record DiagnosticRule(
     string RuleId,
-    string Message);
+    string Message
+    ,
+    Location Location = null
+    );
```

Comment:
No need for new lines.
Also I think this parameter should not be optional, it's always assigned right?

Just
 ```
   string Message,
   Location Location);
```

Reply:
Done